pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.5</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.softwareangels</groupId>
+	<artifactId>store-app</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<name>store-app</name>
+	<description>Demo project for Spring Boot</description>
+	<properties>
+		<java.version>11</java.version>
+		<kotlin.version>1.6.10</kotlin.version>
+		<opencsv>5.5.2</opencsv>
```

Comment:
What is the principle behind putting some version numbers as properties and not others (eg. `org.apache.httpcomponents:httpclient`)?

Reply:
Hi, actually the idea was putting all versions in the properties. I forgot this one.
Will be fix.
Thanks!